doc/source/cluster/running-applications/job-submission/openapi.yml

```diff
@@ -0,0 +1,471 @@
+        500:
+          summary: Job Logs Internal Error
+          description: |
+            An internal error occurred when getting the logs of the job.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        404:
+          summary: Job Not Found
+          description: |
+            The job does not exist.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        400:
+          summary: Wrong Job Type
+          description: |
+            The job was not submitted via the Ray Jobs API, so its logs cannot be retrieved.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+  /api/jobs/{submission_id}/logs/tail:
+    get:
+      summary: Tail Job Logs
+      description: |
+        WebSocket endpoint for tailing the logs of a job 
+        (Not documented in OpenAPI, see 
+        https://docs.ray.io/en/latest/_modules/ray/dashboard/modules/job/sdk.html#JobSubmissionClient.tail_job_logs
+        for example usage).
+      parameters:
+        - name: submission_id
+          in: path
+          description: |
+            The ID of the job to tail the logs of.
+          required: true
+          schema:
+            type: string
+      responses:
+        404:
+          summary: Job Not Found
+          description: |
+            The job does not exist.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        400:
+          summary: Wrong Job Type
+          description: |
+            The job was not submitted via the Ray Jobs API, so its logs cannot be retrieved.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        
+
+
+components:
+  schemas:
+    JobDetails:
+      description: Job data with extra details about its driver and its submission.
+      type: object
+      properties:
+        type:
+          description: The type of job.
+          allOf:
+            - $ref: '#/components/schemas/JobType'
+        entrypoint:
+          title: Entrypoint
+          description: The entrypoint command for this job.
+          type: string
+        job_id:
+          title: Job Id
+          description: >-
+            The job id. An id that is created for every job that is launched in ray.
+            This can be used to fetch data about jobs using ray core apis.
+          type: string
+        submission_id:
+          title: Submission Id
+          description: >-
+            A submission id is an id created for every submission job. It can be used
+            to fetch data about jobs using the job submission apis.
```

Comment:
The antecedent is ambiguous here. Does it refer to id or job. It normally refers to the last antecedent, which is job, and that is incorrect. 

Suggest use "This id can be used to fetch ...